apps/dashboard/src/modules/seller/views/POSInfoView.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="page-container flex flex-column">
+    <div class="page-title">{{ posName }}</div>
+    <div class="flex flex-column gap-5">
+      <div class="flex flex-column md:flex-row gap-5 justify-content-between align-items-stretch w-12">
+        <POSSettingsCard :pos-id="id!" class="flex-1 h-12" />
+        <CardComponent :header="$t('c_posInfo.lastWeekSales')" class="flex-1" >
```

Comment:
Might be nice to also give this card h-12 class to line up with the POSSettingsCard.

Reply:
The CSS doesn't CSS like that, I think I looked at it for about 15 minutes and then decided it was not worth my time. Could be fixed by modifying the CardComponent component, but wasn't worth it for this.